clinica/pipelines/engine.py

```diff
@@ -45,6 +47,16 @@ def get_subject_session_list(input_dir, ss_file=None, is_bids_dir=True, use_sess
             is_bids_dir=is_bids_dir,
             use_session_tsv=use_session_tsv)
 
+    if not os.path.isfile(ss_file):
+        raise ClinicaException(
```

Comment:
`FileNotFoundError` est peut etre plus adaptée (et plus parlante que ClinicaException)

Reply:
J'avais utilisé `ClinicaException` pour ne pas afficher le custom Traceback que tu as développé comme on est dans la situation on l'on connaît l'origine du problème. 

Une solution serait de ne jamais utiliser la classe `ClinicaException` et d'ajouter la classe `ClinicaFileNotFound` pour préciser l'origine de l'exception.

Qu'en penses-tu ?

Reply:
J'avais utilisé `ClinicaException` pour ne pas afficher le custom Traceback que tu as développé comme on est dans la situation on l'on connaît l'origine du problème. 

Une solution serait de ne jamais utiliser la classe `ClinicaException` et d'ajouter la classe `ClinicaFileNotFound` pour préciser l'origine de l'exception.

Qu'en penses-tu ?

Reply:
You are right, your exception is clear enough